packages/components/src/Field/Field.tsx

```diff
@@ -1,49 +1,52 @@
 import React from 'react';
-import { Input, Label, ValidationMessage } from '@marigold/components';
-import { useStyles, system } from '@marigold/system';
+
 import { Exclamation, Required } from '@marigold/icons';
-import b2bTheme from '@marigold/theme-b2b';
+import { useStyles } from '@marigold/system';
+import { ComponentProps } from '@marigold/types';
+
+import { Input } from '../Input';
+import { Label } from '../Label';
+import { ValidationMessage } from '../ValidationMessage';
 
-type FieldProps = {
+export type FieldProps = {
   variant?: string;
   htmlFor: string;
   label: string;
   error?: string;
-};
+} & ComponentProps<'input'>;
 
-export const Field = system<FieldProps, 'input'>(
-  ({
-    variant = 'field',
-    type = 'text',
-    htmlFor,
-    label,
-    error,
-    className = '',
-    ...props
-  }) => {
-    const classNames = useStyles(
-      {
-        variant: `form.${variant}`,
-      },
-      className
-    );
+export const Field: React.FC<FieldProps> = ({
+  variant = 'field',
+  type = 'text',
+  className = '',
+  htmlFor,
+  label,
+  error,
+  ...props
+}) => {
+  const labelClassName = useStyles(
+    {
+      variant: `form.${variant}`,
+    },
+    className
+  );
+  const errorClassName = useStyles({ color: 'colors.red60' });
```

Comment:
I think colors.red60 doesnt work

Reply:
fixed

Reply:
color.red60 funktioniert aucht nicht, wir haben ja nirgends eine Verbindung zur Theme. Das muss man erst bauen (ähnliches Problem wie mit den spaces bei mir). Glaube hilft nur fest die Farbe hier zu hinterlegen.

Reply:
Das müssen wir dann aber in einem separaten Issue lösen. Ich hab das ja nur umgebaut und die Abhängigkeit zu einer bestimmten Theme gelöst.

Reply:
Aber solange bis wir eine passende Lösung haben muss die Farbe ja hinterlegt sein oder nicht? Würde ich schon sagen.

Reply:
Wie gesagt -> ich habe nur Abhängigkeiten, die nicht da sein sollen gelöst. Wenn wir ein Fallback haben wollen, können wir uns gerne in einem separaten Ticket machen. Das hier ist ja nur die Refa.